### ISSUE TITLE
[bug] "Onlook is not enabled" -> "Learn how to enable" invalid link

### ISSUE THREAD

**Issue by mikegreen** [onlook-dev/onlook#303]
#### Describe the bug

Small, but figured it'd be good to know how to enable if/when this exists? 

Clicking the "Learn how to enable" link in the app takes you to:
https://github.com/onlook-dev/onlook?tab=readme-ov-file#using-your-own-react-project
Which does not seem to exist. 

<img width="297" alt="image" src="https://github.com/user-attachments/assets/e859fc2b-c4d5-461f-b98a-a3b971ee6e37">

**Comment by Kitenite** [comment]
Thanks for the catch @mikegreen! We moved that link to [this wiki here](https://github.com/onlook-dev/onlook/wiki/How-to-set-up-my-project%3F). I will add a bug fix for this :)

**Comment by ScubyG** [comment]
@Kitenite, you beat me to updating this issue. I wasn't sure sure if it requires a direct link to the anchor in the article or if the link is was for general documentation.

**Comment by Kitenite** [comment]
@ScubyG could you do the wiki link? I just saw the PR, you work fast!

**Comment by mikegreen** [comment]
Great, thanks folks. Happy to see NextJS in there, starting a new platform and wanted to see how a greenfield project plays in.

**Comment by ScubyG** [comment]
Yes, of course. I will have a look at the app this evening and see if I can find any more issues. I saw the issue on my phone so had to boot up and log in to my trusty old thinkpad which is why I couldn't get to it quicker :)

> @ScubyG could you do the wiki link? I just saw the PR, you work fast!

Yes, of course. I will have a look at the app this evening and see if I can find any more issues. I saw the issue on my phone so had to boot up and log in to my trusty old thinkpad which is why I couldn't get to it quicker :)
